integration-tests/rpc-get-protocol-constants.spec.ts

```diff
@@ -717,8 +822,8 @@ CONFIGS().forEach(({ lib, protocol, rpc }) => {
         max_operation_data_length: 32768,
         max_proposals_per_delegate: 20,
         preserved_cycles: 3,
-        blocks_per_cycle: 128,
-        blocks_per_commitment: 16,
+        blocks_per_cycle: 8192,
```

Comment:
surprised that blocks_per_cycles changes by 64 times

Reply:
yeah, it jumped quite a fair bit